components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/saml/SAML1BearerGrantHandler.java

```diff
@@ -190,12 +190,13 @@ public boolean validateGrant(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
             return false;
         }
 
-        if (assertion == null) {
-            if (log.isDebugEnabled()) {
-                log.debug("Assertion is null, cannot continue");
-            }
-            return false;
-        }
+        //redundant check found by find bugs
+//        if (assertion == null) {
+//            if (log.isDebugEnabled()) {
+//                log.debug("Assertion is null, cannot continue");
```

Comment:
don't keep these commented.
either remove these or keep these uncommented.
format the code as well.